Formula/emacs-plus.rb

```diff
@@ -233,6 +235,17 @@ class EmacsPlus < Formula
     sha256 "f6cd00bcd37be03d8d83c2e590ab7d259ae37632f5954267240ba9439f799359"
   end
 
+  if build.with? "system-appearance-change"
+    unless build.with? "emacs-27-branch"
+      odie "--with-system-appearance-change is supported only when building "\
+           "with --with-emacs-27-branch"
```

Comment:
I understand why it should be `with <option>`, though if feels funny since most options already contain word `with`. Not sure what is the best way to word it haha.

Reply:
I changed the phrasing, since I added a `master`version of the patch.